ci-operator/step-registry/assisted/baremetal/assisted-baremetal-workflow.yaml

```diff
@@ -4,12 +4,13 @@ workflow:
     cluster_profile: packet-assisted
     allow_best_effort_post_steps: true
     pre:
-      - chain: assisted-baremetal-setup
+      - ref: baremetalds-packet-setup
+      - chain: assisted-common
     test:
       - ref: assisted-baremetal-conf
       - ref: assisted-baremetal-test
```

Comment:
`assisted-common-verify` doesn't work out of the box, the conformance tests need to be run on from the host, not from prow. I think it can be fixed later.

Reply:
I agree. It can be fixed later